spec/requests/api/v1/motifs_request_spec.rb

```diff
@@ -1,112 +1,149 @@
 # frozen_string_literal: true
 
-describe "api/v1/motifs requests", type: :request do
-  let!(:organisation) { create(:organisation) }
-  let!(:service) { create(:service) }
+require "swagger_helper"
 
-  describe "GET api/v1/:organisation_id/motifs" do
-    let!(:agent) { create(:agent, service: service, basic_role_in_organisations: [organisation]) }
+describe "RDV authentified API", swagger_doc: "v1/api.json" do
+  path "/api/v1/organisations/{organisation_id}/motifs" do
+    get "Lister les motifs" do
+      tags "Motif"
+      produces "application/json"
+      operationId "getMotifs"
+      description "Renvoie tous les motifs à partir d'une organisation"
 
-    context "when the agent does not belong to the organisation" do
-      let!(:other_orga) { create(:organisation) }
-      let!(:motif) { create(:motif, organisation: other_orga, service: service) }
+      security [{ access_token: [], uid: [], client: [] }]
 
-      it "does not return the motifs list" do
-        get api_v1_organisation_motifs_path(other_orga), headers: api_auth_headers_for_agent(agent)
-        expect(response.status).to eq(200)
-        expect(parsed_response_body["motifs"]).to eq([])
-      end
-    end
+      parameter name: :organisation_id, in: :path, type: :string, description: "Identifiant de l'organisation", example: "1"
+
+      parameter name: "access-token", in: :header, type: :string, description: "Token d'accès (authentification)", example: "SFYBngO55ImjD1HOcv-ivQ"
```

Comment:
Bon ça ne change pas grand chose, mais ça te dirait qu'on utilise que des int pour les ID ?

```suggestion
      parameter name: "access-token", in: :header, type: :integer, description: "Token d'accès (authentification)", example: "SFYBngO55ImjD1HOcv-ivQ"
```

Reply:
yes pour les ID, ça me parait sensé, dans ta suggest par contre tu as mis `access_token`.